apps/web/components/booking/AvailableTimes.tsx

```diff
@@ -132,8 +136,20 @@ const AvailableTimes: FC<AvailableTimesProps> = ({
                 let slotFull, notEnoughSeats;
 
                 if (slot.attendees && seatsPerTimeSlot) slotFull = slot.attendees >= seatsPerTimeSlot;
-                if (slot.attendees && bookingAttendees && seatsPerTimeSlot)
+                if (slot.attendees && bookingAttendees && seatsPerTimeSlot) {
                   notEnoughSeats = slot.attendees + bookingAttendees > seatsPerTimeSlot;
+                }
+
+                const isHalfFull =
+                  slot.attendees && seatsPerTimeSlot && slot.attendees / seatsPerTimeSlot >= 0.5;
+                const isNearlyFull =
+                  slot.attendees && seatsPerTimeSlot && slot.attendees / seatsPerTimeSlot >= 0.83;
+
+                const colorClass = isNearlyFull
+                  ? "text-rose-600"
+                  : isHalfFull
+                  ? "text-yellow-500"
+                  : "text-emerald-400";
```

Comment:
This is duplicated a lot - why dont we move this over to a cva style function to make this a lot more readable? 

<img width="436" alt="CleanShot 2023-07-10 at 10 38 37@2x" src="https://github.com/calcom/cal.com/assets/55134778/b45f2f66-5c82-4491-be27-039e4a1990dc">

<img width="643" alt="CleanShot 2023-07-10 at 10 38 41@2x" src="https://github.com/calcom/cal.com/assets/55134778/95e7e629-e3fc-4ddf-bc1a-5baf8ab236b7">